platforms/web/src/App.tsx

```diff
@@ -51,11 +52,34 @@ function Button({ onClick, imagePath, alt, state }: ButtonProps) {
 }
 
 function App() {
-    const { ref, isWysiwygReady, wysiwyg, formattingStates, debug } =
+    const [enterToSend, setEnterToSend] = useState(true);
+
+    const inputEventProcessor = (
+        e: WysiwygInputEvent,
+        wysiwyg: Wysiwyg,
+    ): WysiwygInputEvent | null => {
+        if (e instanceof ClipboardEvent) {
+            return e;
+        } else if (enterToSend && e.inputType === 'insertParagraph') {
+            console.log(`SENDING: ${wysiwyg.content()}`);
+            e.preventDefault();
+            e.stopPropagation();
+            wysiwyg.actions.clear();
+            return null;
+        }
+        return e;
+    };
+
+    const { ref, isWysiwygReady, formattingStates, wysiwyg, debug } =
         useWysiwyg({
             isAutoFocusEnabled: true,
+            inputEventProcessor,
         });
 
+    const onEnterToSendChanged = () => {
+        setEnterToSend(!enterToSend);
```

Comment:
When using a value from the state to create a new state, we should use directly the function parameter from `setState`

`setEnterToSend(_enterToSend => !_enterToSend)`